2.segment_images/scripts/2.segment_whole_organoids.py

```diff
@@ -205,17 +263,21 @@
         plt.show()
 
 
-# In[7]:
+# In[8]:
 
 
-# Use butterworth FFT filter to remove high frequency noise :)
-imgs = skimage.filters.butterworth(
-    cyto,
-    cutoff_frequency_ratio=0.5,
-    high_pass=False,
-    order=1,
-    squared_butterworth=True,
-)
+# # Use butterworth FFT filter to remove high frequency noise :)
+for i in range(cyto.shape[0]):
+    cyto[i, :, :] = skimage.filters.butterworth(
+        cyto[i, :, :],
+        cutoff_frequency_ratio=0.05,
+        high_pass=False,
+        order=1,
+        squared_butterworth=True,
+    )
+
+# add a guassian blur to the image
```

Comment:
Why do we add guassian blur to the image? Consider answering within the code comment. Also - could this impact the data integrity at all?